nodeadm/internal/kubelet/config.go

```diff
@@ -79,6 +79,10 @@ type kubeletConfig struct {
 	RegisterWithTaints       []v1.Taint                       `json:"registerWithTaints,omitempty"`
 }
 
+type loggingConfiguration struct {
+	Verbosity logsapi.VerbosityLevel `json:"verbosity"`
```

Comment:
is `VerbosityLevel` just an alias for an integer type? Unless we get some validation during serialization with this, we should just use `int` here.

Reply:
Ah it's in component-base.

Thinking about this more -- if we write out the entire default `logsapi.LoggingConfiguration`, wouldn't kubelet still apply its own defaults on top of that? is this change actually necessary?

Reply:
it is yea, that's fair to change

Reply:
there's [one hairy field](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-base/logs/api/v1/types.go#L112C2-L112C39), plus i think it's no issue to just minimize the output json with this change